### ISSUE TITLE
Script fails instead of waiting for service to become stable

### ISSUE THREAD

**Issue by mfboulos** [aws-actions/amazon-ecs-deploy-task-definition#5]
Previous discussion on this issue in #2 

I've got a deployment script that has been successfully deploying my application to ECS. However, this GHA script errors out with the following debug logs:

```
##[debug]Evaluating condition for step: 'Deploy Amazon ECS task definition'
##[debug]Parsing expression: <success()>
##[debug]Evaluating: success()
##[debug]Evaluating success:
##[debug]=> true
##[debug]Result: true
##[debug]Starting: Deploy Amazon ECS task definition
##[debug]Loading inputs
##[debug]Evaluating: steps.task-def.outputs.task-definition
##[debug]Evaluating Index:
##[debug]..Evaluating Index:
##[debug]....Evaluating Index:
##[debug]......Evaluating steps:
##[debug]......=> Object
##[debug]......Evaluating String:
##[debug]......=> 'task-def'
##[debug]....=> Object
##[debug]....Evaluating String:
##[debug]....=> 'outputs'
##[debug]..=> Object
##[debug]..Evaluating String:
##[debug]..=> 'task-definition'
##[debug]=> '/home/runner/work/_temp/task-definition-27961zBZiXxjdRic.json'
##[debug]Result: '/home/runner/work/_temp/task-definition-27961zBZiXxjdRic.json'
##[debug]Loading env
Run aws-actions/amazon-ecs-deploy-task-definition@v1
  with:
    task-definition: /home/runner/work/_temp/task-definition-27961zBZiXxjdRic.json
    service: ob-service
    cluster: ob-cluster
    wait-for-service-stability: true
  env:
    AWS_ACCESS_KEY_ID: ***
    AWS_SECRET_ACCESS_KEY: ***
    AWS_DEFAULT_REGION: us-west-1
    AWS_REGION: us-west-1
##[debug]Registering the task definition
::set-output name=task-definition-arn,::arn:aws:ecs:us-west-1:***:task-definition/ob-task-definition:4
##[debug]='arn:aws:ecs:us-west-1:***:task-definition/ob-task-definition:4'
##[debug]Updating the service
##[debug]Waiting for the service to become stable
##[error]Resource is not in the state servicesStable
##[error]Node run failed with exit code 1
##[debug]Finishing: Deploy Amazon ECS task definition
```

I tracked the deployment process, and this is what I found:

* The script only waits a couple seconds to error out, even with `wait-for-service-stability: true`
* The service deployed to is the only service in the cluster, and it's active
* The service has a single deployment

I took a look at the event logs in the service, and found this too:

```
2019-11-17 12:43:39 -0800
service ob-service has started 1 tasks: task d0bbf143-4308-4c3f-ac1f-a7fc2b9f7b13.
```

followed by

```
2019-11-17 12:44:21 -0800
service ob-service has reached a steady state.
```

Is there an option to customize the wait time for stability? And if not, what is that timeframe supposed to look like?

**Comment by clareliguori** [comment]
I was able to reproduce this when my creds didnt have the right `ecs:DescribeServices` permissions. #4 will make this missing permission more obvious in the logs, since it calls DescribeServices prior to the deployment.

Are you scoping down by resource and have you opted into the long ARN format?

**Comment by mfboulos** [comment]
I did not actively setup anything to scope down by resource or opt into any ARN format. I'm mostly using what was given to me by the presets on Github Actions and learning along the way.

With that said, after adding that permission to the IAM user running the deployment, it worked! Having the logs describe that a little better would definitely be useful for anyone else who runs into this. Thanks a ton.

**Comment by clareliguori** [comment]
Fixed by #4
